R/tab.R

```diff
@@ -8,19 +8,20 @@
 #' The convenience functions `tab1()` and `tab2()` are inspired by functions of the same name in Stata. They allow rapid tabulation of a set of variables. `tab1()` creates one-way tables of frequencies for each listed variable. `tab2()` creates two-way tables of frequencies for all listed variable combinations.
 #'
 #' @usage
-#' tab(x, ..., m = TRUE)
+#' tab(x, ..., m = TRUE, sort = FALSE)
 #'
-#' ta(x, ..., m = TRUE)
+#' ta(x, ..., m = TRUE, sort = FALSE)
 #'
-#' ftab(x, ..., m = TRUE)
+#' ftab(x, ..., m = TRUE, sort = FALSE)
 #'
-#' tab1(x, ..., m = TRUE)
+#' tab1(x, ..., m = TRUE, sort = FALSE)
 #'
 #' tab2(x, ..., m = TRUE)
 #'
 #' @param x A vector, data.frame, or tibble.
 #' @param ... A comma separated list of unquoted variable names or positions. Select helpers from [dplyr](https://dplyr.tidyverse.org/reference/select.html) and [tidyselect](https://rdrr.io/cran/tidyselect/man/select_helpers.html) are supported.
 #' @param m If `TRUE` (the default), missing values are reported.
+#' @param sort If `TRUE` (default is `FALSE`), will show the largest groups at the top. Sorting is not implemented for two-way tables.
```

Comment:
Thank you for this PR. It might be worth mentioning here that the default behavior is to sort by value and that `sort = TRUE` changes this behavior to sort by frequency.

Reply:
done